### DISCUSSION TITLE
Why occur error at this expression?

### DISCUSSION THREAD

**Discussion by megaTRX** [community/community#51432] [Programming Help]
### Select Topic Area

Question

### Body

int main(void) {
	int a = 10;
	printf("a는 %d\n", a);
	a++;
	printf("a는 %d\n", a);
	return 0;
}

**Comment by leanczo** [comment]
This code should compile and run just fine, as long as you've included the standard C input/output library (#include <stdio.h>) at the top of the source code file.

Source: ChatGPT.

Likewise, if you provide us with more information about the error, we could give you a better answer.

**Comment by Anmol-Baranwal** [comment]
> Although, this is not a correct place to ask such questions.

The output:

```c++
a는 10
a는 11
```

Error:
In this particular program, the standard header file "stdio.h" is required for the printf() function to work properly. Without including this header file, you may encounter a warning or error message when compiling the program.

**Reply by Anmol-Baranwal** [reply]
@megaTRX 
@JDiSolutions 
I wanted to follow up and ask if you have any additional questions or concerns about my previous response on GitHub. If you have no further inquiries, would you kindly consider marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @megaTRX ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
